src/index.html

```diff
@@ -1,13 +1,48 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8" >
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <title>Product cards</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+    <link rel="stylesheet" href="./style.css" >
+  </head>
+  <body>
+    <div class="card" data-qa="card">
+      <div class="card__img-imac"></div>
+      <div class="card__about">
+        APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+      </div>
+      <div class="card__id">Product code: 195434</div>
+      <div class="card__rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <div class="card__reviews">Reviews: 5</div>
+      </div>
+      <div class="card__price">
+        <span class="card__text">Price:</span>
+        <span class="card__value">$2,199</span>
+      </div>
+      <div class="card__buy">
+        <a
+          href="#buy"
+          class="card__button"
+          data-qa="hover"
+        >
+          BUY
+        </a>
+      </div>
```

Comment:
it's a redundant `<div>` wrapper
```suggestion
        <a
          href="#buy"
          class="card__button"
          data-qa="hover"
        >
          BUY
        </a>
```

Reply:
done